.github/workflows/build_and_push.yml

```
@@ -66,8 +66,8 @@ jobs:
 
           tags: |
             type=ref,enable=true,priority=600,prefix=pr-,suffix=,event=pr
+            type=ref,enable=true,priority=600,prefix=,suffix=,event=tag
             type=sha,format=short,prefix=commit-,event=pr
```

```suggestion
            type=ref,enable=true,priority=600,prefix=,suffix=,event=push
            type=sha,format=short,prefix=commit-,event=pr
```

> Could we stick to event=push and change to type=ref?

@m-kuhn something like this ?

BTW I don't think the `event=pr` is used that much.